assets/xml/objects/gameplay_keep.xml

```diff
@@ -1468,7 +1483,7 @@
         <DList Name="gameplay_keep_DL_085418" Offset="0x85418" />
         <DList Name="gameplay_keep_DL_085490" Offset="0x85490" />
         <Blob Name="gameplay_keep_Blob_085510" Size="0x130" Offset="0x85510" />
-        <Blob Name="gameplay_keep_Blob_085640" Size="0x3A30" Offset="0x85640" />
+        <Blob Name="gameplay_keep_Blob_085640" Size="0x10" Offset="0x85640" />
 
         <!-- Zora Elegy of Emptiness Shell -->
         <DList Name="gElegyShellZoraDL" Offset="0x89070" /> <!-- Original name is "pzs_zo_model" -->
```

Comment:
So what happens with all that data between `Offset="0x85650"` and `Offset="0x89070"`?

Reply:
They are Vtxs, which we usually don't put on the xmls unless we really need them (like cross referencing them during extraction)